daemon/bpf.sha

```diff
@@ -1,2 +1,2 @@
-GO_BINDATA_SHA1SUM=3fc90f93eb8197014a31fdd600944d44de9d91e6
-GO_VERSION_USED=go1.9.2
+GO_BINDATA_SHA1SUM=9dd2c734136386861acd3ff8ff5024721d42cba4
+GO_VERSION_USED=go1.8.3
```

Comment:
This isn't my area of expertise, but should we require that `daemon/bpf.sha` be compiled with a certain version of golang? This could be added as a runtime test. @aanm is this a hard requirement?

Reply:
We discussed this before and it's not really a problem. We have this for visibility but there is no reason why do enforce it. We have been using different versions for a while without any issues.

Reply:
In fact, I've seen it flip flop between 1.9.2 and 1.8.x today :(